Header header
geometry_msgs/PoseWithCovariance pose
geometry_msgs/Polygon			 tabletop
string							 table_id
string							 topological_node
